sample/src/main/java/org/walletconnect/example/MainActivity.kt

```diff
@@ -1,11 +1,103 @@
 package org.walletconnect.example
 
-import androidx.appcompat.app.AppCompatActivity
+import android.graphics.drawable.ColorDrawable
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import org.walletconnect.example.databinding.ActivityMainBinding
+import org.walletconnect.example.extension.getCurrentDestination
+import org.walletconnect.example.extension.getNavController
+import org.walletconnect.example.wallet.WalletViewModel
+import org.walletconnect.example.wallet.ui.SessionProposalDialog
+import org.walletconnect.example.wallet.ui.ShowSessionProposalDialog
+import org.walletconnect.example.wallet.ui.ToggleBottomNav
 
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private val viewModel: WalletViewModel by viewModels()
+    private var sessionProposalDialog: SessionProposalDialog? = null
+
```

Comment:
add this variable here
```suggestion
private val navHostFragment by lazy { supportFragmentManager.findFragmentById(R.id.nav_host_container) as NavHostFragment }
```